index.js

```diff
@@ -60,6 +60,8 @@ function resolveUrlLoader(content, sourceMap) {
 
   // incoming source-map
   var sourceMapConsumer, contentWithMap, sourceRoot;
+  // NOTE: source-map from the less-loader comes as string
+  sourceMap = (typeof sourceMap === 'string') ? JSON.parse(sourceMap) : sourceMap;
```

Comment:
Also feel free to pass `handleException` a more descriptive label and/or message.